pkg/shifuctl/cmd/install/install.go

```diff
@@ -0,0 +1,167 @@
+package install
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/spf13/cobra"
+)
+
+var InstallCmd = &cobra.Command{
+	Use:   "install",
+	Short: "install shifu and its dependencies",
+	Run: func(cmd *cobra.Command, args []string) {
+		install()
+	},
+}
+
+// install installs shifu and its dependencies.
+func install() {
+	// start a spinner
+	s := spinner.New(spinner.CharSets[35], 200*time.Millisecond)
+	s.Prefix = "\033[1;33mChecking Shifu prerequisites...\033[0m"
+	s.Suffix = "\n"
+	s.Start()
+
+	prompt := ""
+	prompt = prompt + verifyDockerInstallation()
+	prompt = prompt + verifyDockerHubConnection()
+	prompt = prompt + verifyKubectlInstallation()
+	prompt = prompt + verifyHelmInstallation()
+	prompt = prompt + verifyKubernetesCluster()
+	// If there's no error, then prompt is empty
+	if prompt == "" {
+		prompt = "\033[1;32mPrerequisite met, automatically installing Shifu!\033[0m\n"
+	}
+	fmt.Println(prompt)
+
+	s.Stop()
+
+	if err := installShifu(); err != nil {
+		fmt.Println("\033[1;31mError installing Shifu\033[0m, Error: ", err)
```

Comment:
use zlog?

Reply:
This is a command line utility, fmt should be sufficient?